### DISCUSSION TITLE
Several Source to one Destination?

### DISCUSSION THREAD

**Discussion by sapient369** [radinshayanfar/TGCopyBot#12] [Q&A]
Is it possible to add several source(group/channel) to one destination?

**Comment by radinshayanfar** [comment]
Hi @sapient369 
Yes, you can select which chats you want the handler to process. It is basically done in the `new_message_handler`. Here for example: https://github.com/radinshayanfar/TGCopyBot/blob/ecb0b5eb1bc9f5c5fc1a402d5b8c6bf774149c34/app/main.py#L63
So edit this to reflect your needs.
